components/pedigree/resources/src/main/resources/pedigree/view/personPlaceholderVisuals.js

```diff
@@ -13,6 +13,8 @@ define([
     var PersonPlaceholderVisuals = Class.create(PersonVisuals, {
         initialize: function($super, node, x, y) {
             $super(node,x,y);
+            // Remove the unused highlight box that gets generated in the parent
+            this._highlightBox && this._highlightBox.remove(); 
```

Comment:
I think a better approach is to never create the highlightBox in the first place, by overwriting the `setHighlightBox()` method to be `this._highlightBox && this._highlightBox.remove();` instead of what it is now